udata/core/site/views.py

```diff
@@ -73,9 +73,30 @@ def map():
     return theme.render('site/map.html')
 
 
+def get_export_url(model):
+    slug = current_app.config['EXPORT_CSV_DATASET_INFO']['slug']
+    dataset = Dataset.objects.get_or_404(slug=slug)
+    resource = None
+    for r in dataset.resources:
+        if r.extras.get('csv-export:model', '') == model:
+            resource = r
+            break
+    if not resource:
+        abort(404)
+    return resource.url
+
+
 @blueprint.route('/datasets.csv', cors=True)
 def datasets_csv():
     params = multi_to_dict(request.args)
+    # redirect to EXPORT_CSV dataset if feature is enabled and no filter set
```

Comment:
"no filter set" ça ne veut pas dire qu'il y a n'y a pas de filtre. MAYBE "and no filter is set" ou "filter is not set"